readme.md

```diff
@@ -30,16 +30,27 @@ The output is a JSON object in the form:
 
 ```json
 {
-    "bundle_name": {
-        "asset_kind": "/public/path/to/asset"
-    }
+    "chunks": {
+      "bundle_name": {
+          "asset_kind": "/public/path/to/asset"
+      }
+    },
+    "images": [{
+        "name": "hello.svg",
+        "path": "/public/path/to/asset/hello.svg"
+      }
+    ]
```

Comment:
@kossnocorp my idea was to separate assets by type. I am using extracted `images` array to resolve URLs to images when rendering on server-side. I guess having more generic `assets` would work as well. Just less data to iterate over when looking up a URL.